### ISSUE TITLE
Support abbreviation markup

### ISSUE THREAD

**Issue by StephDriver** [BirkbeckCTP/janeway#4209]
Journal articles often include abbreviations:
- subject/industry specific e.g. the name of a governing body
- article specific as defined by the authors which should be listed in the article
- units e.g. `min`, `mL`, `cm`
- associated with academic publishing: page number, issue numbers, abbreviations used within citation formats.

Abbreviation markup is not in WCAG 2.2 AA, but in AAA, so adding this issue to be reviewed in Phase 2 of the A11y project.

**Comment by StephDriver** [comment]
Issues identified in #4194 

- Sample 2, item 40. abbreviation not marked up. This is just written as text: but “there is no reference (in the strong sense) to a specific moment” (p. 6). This seems to us to be a more adequate but the p. is an abbreviation for page, probably should be marked so that it gets read out as page, to avoid confusion.
- Sample 2, item 52. DOI is read out as a word to rhyme with boy, this needs to be marked up as an abbreviation and to be read as D O I
- Sample 4, item 16. the abbreviation IRB is not read out as "irb" - mark up as an abbreviation? to make sure letters are read out.
- Sample 4, item 17. markup for abbreviations missing when abbreviations are used in the body of text
- Sample 4, item 27. A 2.54-cm-wide cut not read correctly - cm not read as centimetres, but as c m I think this is to do with the use of hyphens, as later in the same paragraph, 2.54 cm is read as "two dot five four centimetres"
- Sample 4, item 28. similarly, 3-min is read as "3 min" rather than "3 minutes"
- Sample Other, item 14. abbreviations such as e.g. and i.e. are automatically read out as for example and such that respectively. While not a problem here, might be worth considering highlighting this behaviour to editors.
- Sample Other, item 18. abbreviations that can be read as words are read as words - need to mark them up as abbreviations where-ever used.
